contents/docs/cdp/index.md

```diff
@@ -1,32 +1,18 @@
 ---
-title: Customer data platform
-sidebarTitle: Overview
-sidebar: Docs
+title: Data pipeline
 showTitle: true
 ---
 
-Data connections include sources (data in) and destinations (data out), and are part of PostHog's CDP (officially coming soon). They allow anyone to extend and customize PostHog in order to better fit their needs.
+Data pipeline includes apps and batch exports. Apps extend PostHog's functionality by filtering and transforming data, as well as sending it to destinations. [Batch exports](/docs/cdp/batch-exports) reliably send data to a destination on a schedule.
```

Comment:
In the new UI we're avoiding the term "apps" and instead talking about data pipeline transformations (filtering or modifying data) and destinations (either webhooks or batch exports)